specification/common-types/resource-management/v4/types.json

```diff
@@ -0,0 +1,670 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "3.0",
```

Comment:
this should be 4.0

Reply:
Ah, indeed. Thanks for caching this. Everyone focused on the changes, forgot about this.